renderdoc/driver/gl/gl_common.h

```diff
@@ -56,11 +56,17 @@ struct GLWindowingData
 };
 
 #elif ENABLED(RDOC_LINUX)
+
+#ifdef RENDERDOC_SUPPORT_GL
```

Comment:
In the gl_hooks_linux.cpp there are a few conflicting functions (like GetRealGLFunctions, etc..) so to have bot GL and GLES enabled first we would need to sort that issue out. Also adding extra logic to correctly detect if we are using GL or GLES in runtime. For the current PR I wanted to keep it as clean as possible and discuss what would be the best way to merge GL and GLES together.